tasks/main-blivet.yml

```diff
@@ -101,15 +31,39 @@
     name: "{{ package_info.packages }}"
     state: present
 
-- name: manage the pools and volumes to match the specified state
-  blivet:
-    pools: "{{ _storage_pools }}"
-    volumes: "{{ _storage_volumes }}"
-    use_partitions: "{{ storage_use_partitions }}"
-    disklabel_type: "{{ storage_disklabel_type }}"
-    safe_mode: "{{ storage_safe_mode }}"
-    diskvolume_mkfs_option_map: "{{ __storage_blivet_diskvolume_mkfs_option_map|d(omit) }}"
-  register: blivet_output
+- command: 'systemctl list-units --all --plain --full --no-legend -t service'
+  register: storage_all_services
+  changed_when: false
+
+- set_fact:
+    storage_cryptsetup_services: "{{ storage_all_services.stdout_lines|map('regex_search', '^systemd-cryptsetup@\\S+.service')|select('string')|list }}"
+
+- block:
+  - name: Mask the systemd cryptsetup services
+    systemd:
+      name: "{{ item }}"
+      masked: yes
+    loop: "{{ storage_cryptsetup_services }}"
+
+  - name: manage the pools and volumes to match the specified state
+    blivet:
+      pools: "{{ storage_pools|default([]) }}"
+      volumes: "{{ storage_volumes|default([]) }}"
+      use_partitions: "{{ storage_use_partitions }}"
+      disklabel_type: "{{ storage_disklabel_type }}"
+      pool_defaults: "{{ storage_pool_defaults }}"
+      volume_defaults: "{{ storage_volume_defaults }}"
+      safe_mode: "{{ storage_safe_mode }}"
+      diskvolume_mkfs_option_map: "{{ __storage_blivet_diskvolume_mkfs_option_map|d(omit) }}"
+    register: blivet_output
+  rescue:
+    - fail:
```

Comment:
If you want to check the result of the failure in tests, you'll have to "re-raise" the error - see https://richm.github.io/how-to-catch-and-reraise-errors-in-ansible

Reply:
Fixed in latest version, although tests were passing as it was.

Reply:
> Fixed in latest version, although tests were passing as it was.

Then you probably aren't trying to `rescue` and check the error in a calling context.